packages/yoga/src/Tag/native/Informative.jsx

```diff
@@ -81,14 +82,15 @@ const TagInformative = ({
 
 TagInformative.propTypes = {
   /** style the tag following the theme (success, informative, attention) */
-  variant: oneOf(['success', 'informative', 'attention']).isRequired,
+  variant: oneOf(['', 'success', 'informative', 'attention']),
```

Comment:
What do you think about having a `default` variant instead?

```suggestion
  variant: oneOf(['default', 'success', 'informative', 'attention']),
```
Then below
```diff
TagInformative.defaultProps = {
--  variant: '',
++  variant: 'default',
```
Same suggestion for web